tests/MiniCover.ApprovalTests/_ApprovalTests/Reporters/TravisCIReporter.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using ApprovalTests.Core;
+
+namespace ApprovalTests.Reporters
+{
+    public class TravisCIReporter : IEnvironmentAwareReporter
```

Comment:
This file was inside _ApprovalTests folder, but it doesn't exist on ApprovalTests library.

Reply:
Yeah I m going to remove it